src/entry.lisp

```diff
@@ -13,7 +13,8 @@
    #:*global-environment*
    #:entry-point                        ; FUNCTION
    #:expression-entry-point             ; FUNCTION
-   #:file-entry-point                   ; FUNCTION
+   #:compile-coalton                    ; FUNCTION
```

Comment:
nit: maybe consider renaming these functions?

Reply:
I agree these names are not quite right. I think there will be better focus when the calling patterns from swank and emacs have been run through. Ok to leave as is (for the moment)?

Reply:
yes